extensions/sql-database-projects/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "sql-database-projects",
+  "displayName": "SQL Server Database Projects",
+  "description": "SQL Server Database Projects for Azure Data Studio designing and deploying database schemas.",
+  "version": "0.1.0",
+  "publisher": "Microsoft",
+  "preview": true,
+  "engines": {
+		"vscode": "^1.30.1",
+		"azdata": ">=1.12.0"
+  },
+  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/extensions/import/Microsoft_SQL_Server_Import_Extension_and_Tools_Import_Flat_File_Preview.docx",
```

Comment:
normally we just point to [https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/LICENSE.txt](https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/LICENSE.txt) for the license. Is this one pointing at the import license for any specific reason?

Reply:
Copy-pasta from schema-compare

Reply:
oops should also fix this in dacpac and schema compare extensions. Copy-pasta from import extension